cask-repair

```diff
@@ -9,6 +9,7 @@ readonly caskroom_taps=(homebrew-cask homebrew-versions homebrew-fonts homebrew-
 readonly caskroom_taps_dir="$(brew --repository)/Library/Taps/caskroom"
 readonly user_agent=(--user-agent 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10) http://caskroom.io')
 readonly hub_config="${HOME}/.config/hub"
+readonly git_username="$(awk '/user:/{print $(NF)}' "${hub_config}" | head -1)"
```

Comment:
This can’t be here so soon. At this point we haven’t yet confirmed the user has `hub` installed. This will output an `awk` error if they don’t.

We can add it in a better place or redirect stderr to `/dev/null`. Perhaps the latter is better — keeping these constants at the top is pretty useful.